linkml/generators/prefixmapgen.py

```diff
@@ -64,11 +66,32 @@ def end_schema(self, base: Optional[str] = None, output: Optional[str] = None, *
             context[k] = v
         for k, v in self.slot_class_maps.items():
             context[k] = v
+
         if output:
-            with open(output, 'w') as outf:
-                outf.write(as_json(context))
+            output_ext = output.split(".")[-1]
+
+            if output_ext == "tsv":
+                mapping: Dict = {}
+                for prefix in sorted(self.emit_prefixes):
+                    mapping[prefix] = self.namespaces[prefix]
+
+                with open(output, 'w') as outf:
+                    writer = csv.writer(outf, delimiter='\t')
+                    for key, value in mapping.items():
+                        writer.writerow([key, value])
+            else:
+                with open(output, 'w') as outf:
+                    outf.write(as_json(context))
         else:
-            print(as_json(context))
+            if self.format == "tsv":
+                mapping: Dict = {}  # prefix to IRI mapping
+                for prefix in sorted(self.emit_prefixes):
+                    mapping[prefix] = self.namespaces[prefix]
+                
+                for key, value in mapping.items():
+                    print(key, value, sep='\t')
```

Comment:
is this right?

Reply:
ah I see, it's just a 2 column tsv so this is OK - but why not use this same simple form when writing to a file?

Reply:
I wanted to use `csv.writer()` to make sure the data is being correctly formatted and written to a tsv file rather than stringing the `key` and `value` together with a `\t` separator.